react-frontend/src/profile/ListingCollection.js

```diff
@@ -1,10 +1,40 @@
-import React from "react";
+import React, {useState} from "react";
+import Notify from '../components/ErrorNotification';
 import PropTypes from "prop-types";
 import { Link } from "react-router-dom";
+import axios from "axios";
+import editHover from "../assets/edit-hover.png";
+import editNeutral from "../assets/edit-neutral.png";
+import deletePic from "../assets/delete-button.png";
 
 function ListingCollection(props) {
+  const [showNotification, setShowNotification] = useState(false);
+  const [isSuccessful, setIsSuccessful] = useState(false);
+  const [bookmarks, setBookmarks] = useState(props.bookmarks);
+  const [notificationMsg, setNotificationMsg] = useState('');
   console.log(props.bookmarks);
 
+  const handleDeleteListing = async (listingID) => {
+    const confirmed = window.confirm("Are you sure you want to delete this listing?");
+    
+    if (confirmed) {
+      try {
+        console.log("listingID deleting: ", listingID);
+        await axios.delete(process.env.REACT_APP_BACKEND_LINK + `/listings/${listingID}`);
+        setBookmarks((prevBookmarks) => prevBookmarks.filter(bookmark => bookmark.listingID !== listingID));
+        setIsSuccessful(true);
+        setNotificationMsg('Successfully deleted');
+        setShowNotification(true);
```

Comment:
instead of using multiple lines for multiple calls every time you want to show a notification, do you think this could be combined into one function call?